recipe/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-./configure --prefix=$PREFIX
+./configure --prefix=$PREFIX PERL='/usr/bin/env perl'
```

Comment:
Not sure if that change is desirable b/c you are relying on the system's `PERL` instead of `conda`'s.